### PR TITLE
TELCODOCS-2002 -  Telco Day 2 Operations - Observability - ContentX improvements

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-2002](https://issues.redhat.com//browse/TELCODOCS-2002) -  Telco Day 2 Operations - Observability - ContentX improvements

Version(s): 

- 4.18 
- 4.17
- 4.16
 

Issue:https://issues.redhat.com/browse/TELCODOCS-2002

Link to docs preview: [Observability in OpenShift Container Platform](https://86262--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/day_2_core_cnf_clusters/observability/telco-observability)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by elenagerman** [comment]
LGTM

File: edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-observability"]
+= Observability in {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: telco-observability
+:imagesdir: images
+
+toc::[]
+
+{product-title} generates a large amount of data such as performance metrics and logs from both the platform and the the workloads running on it.
```
Comment (skrthomas) [inline edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc:10]:
```suggestion
{product-title} generates a large amount of data, such as performance metrics and logs, from both the platform and the the workloads running on it. 
```

File: modules/telco-observability-workload-monitoring.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+<1> Use endpoints to define workload metrics. 
+
+. Apply the `ServiceMonitor` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f monitoringServiceMonitor.yaml
+----
+
+Prometheus scrapes the path `/metrics` by default, however you can define a custom path in the spec.
```
Comment (skrthomas) [inline modules/telco-observability-workload-monitoring.adoc:69]:
```suggestion
Prometheus scrapes the path `/metrics` by default, however you can define a custom path in the `spec.` fields. 
```

File: modules/telco-observability-key-performance-metrics.adoc [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+|===
+
+== Recommendations for storage of metrics
+
+Out of the box, Prometheus does not back up saved metrics with persistent storage. 
+If you restart the Prometheus pods, all metrics data are lost. 
+You should configure the monitoring stack to use the back-end storage that is available on the platform. 
+To meet the high IO demands of Prometheus you should use local storage.
+
+For Telco core clusters, you can use the `LocalStorage` Operator for persistent storage for Prometheus.
```
Comment (skrthomas) [inline modules/telco-observability-key-performance-metrics.adoc:136]:
```suggestion
For Telco core clusters, you can use the Local Storage Operator for persistent storage for Prometheus. 
```

Comment (skrthomas) [reply modules/telco-observability-key-performance-metrics.adoc:136]: The name of the Operator, afaict, doesn't use backticks. If there's a CR for the Operator, then that would be in backticks for sure, but this reads as the Operator, not the CR for the Operator.

**Review by skrthomas — COMMENTED** [review]
Nice layout overall. Only a few comments for you :) Looking great!

File: edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc (hunk 1 above)
Comment (slovern) [reply edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc:10]: Thanks for the review @skrthomas

File: modules/telco-observability-workload-monitoring.adoc (hunk 2 above)
Comment (slovern) [reply modules/telco-observability-workload-monitoring.adoc:69]: Thanks, I've made this a bit clearer, I think.

File: modules/telco-observability-key-performance-metrics.adoc (hunk 3 above)
Comment (slovern) [reply modules/telco-observability-key-performance-metrics.adoc]: Thanks again @skrthomas  - very useful comments

File: modules/telco-observability-alerting.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+:_mod-docs-content-type: PROCEDURE
+[id="telco-observability-alerting_{context}"]
+
+= Alerting
+
+{product-title} includes a large number of alert rules, which can change from release to release. 
+
+== Viewing default alerts
+
+To review all the alert rules in a cluster, you can run the following command:
```
[the remark this answers is not in the document]
Comment (slovern) [reply modules/telco-observability-alerting.adoc:14]: Yep, that makes sense

File: modules/telco-observability-alerting.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+[source,terminal]
+----
+$ oc get cm -n openshift-monitoring prometheus-k8s-rulefiles-0 -o yaml
+----
+
+Rules can include a description and provide a link to additional information and mitigation steps.
```
[the remark this answers is not in the document]
Comment (slovern) [reply modules/telco-observability-alerting.adoc:20]: Yes this should be treated as output. Thansk @stevsmit

**Pull request merged**